### DISCUSSION TITLE
Sync db for multiple devices

### DISCUSSION THREAD

**Discussion by stoneubi** [fast4x/RiMusic#1749] [Ideas]
I do have multiple devices with rimusic installed.

I also do have my own nextcloud instance.

Feature request: 
It would be awesome if within the app a synchronization of the database can be enabled to sync the database to a nextcloud shared folder and to keep all devices synched and having the same settings, songs, playlist.

Thank you very much 
Stoneubi

**Comment by fast4x** [comment]
This feature would require login with authentication, currently there is no login functionality.

**Reply by darkza** [reply]
now it is...
